light-mode-added

Lavanya-Sathya opened the pull request:
Fixes #56 

Added Switch between dark-light mode.

Lavanya-Sathya committed:
light-mode-added

miljkovicjovan commented:
Hey I haven't had much time and I haven't checked the code yet but one change I would like you to make is that the dark and light mode should be separated from the Show Confirmations category into it's own Dark / Light Mode category with the switch next to it

Lavanya-Sathya committed:
switch-mode-updated

Lavanya-Sathya commented:
Hey, I have removed the dark and light mode from the Show Confirmations category  and added  it next to setting button, let me know if this meets the requirement or I have to make any further changes :)

miljkovicjovan commented:
thanks @Lavanya-Sathya for your contribution
